test/functionalTest/orionWS.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import websocket
```

Comment:
I understand that this is a dependency that needs to be installed using pip. Please, include the installation command to do that as a comment just above the import lines.